kas-installer.env.example

```diff
@@ -18,10 +18,6 @@ IMAGE_REPOSITORY_PASSWORD=<value>
 # [optional] Zone the cluster is in
 # Default: us-east-1
 #
-# Note: Currently setting this to anything other than us-east-1 will
-# result in an error when trying to create a Kafka Instance using the
-# managed_kafka.sh script, since the fleet manager doesn't accept any
-# other region.
 #REGION=<value>
```

Comment:
When someone overrides this, and say set it to `eu-west-1`, things should still work for them as the fleet manager supports multiple cloud provider and region so the comment was only relevant because kas-installer didn't support overriding the supported cloud providers.
@MikeEdgar